Src/FluentAssertions/Primitives/StringAssertions.cs

```diff
@@ -669,6 +669,93 @@ public AndConstraint<StringAssertions> ContainEquivalentOf(string expected, stri
         }
 
         /// <summary>
+        /// Asserts that a string contains all values present in <paramref name="values"/>.
+        /// </summary>
+        /// <param name="values">
+        /// The values that should all be present in the string
+        /// </param>
+        /// <param name="because">
+        /// A formatted phrase as is supported by <see cref="string.Format(string,object[])" /> explaining why the assertion 
+        /// is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
+        /// </param>
+        /// <param name="becauseArgs">
+        /// Zero or more objects to format using the placeholders in <see cref="because" />.
+        /// </param>
+        public AndConstraint<StringAssertions> ContainAll(IEnumerable<string> values, string because = "", params object[] becauseArgs)
+        {
+            ThrowIfValuesNullOrEmpty(values);
+
+            var missing = values.Where(v => !Contains(Subject, v, StringComparison.Ordinal)).ToArray();
+            Execute.Assertion
+                .ForCondition(values.All(v => Contains(Subject, v, StringComparison.Ordinal)))
+                .BecauseOf(because, becauseArgs)
+                .FailWith("Expected string {0} to contain strings: {1}{reason}.", Subject, missing);
```

Comment:
Use the phrase `Expected {context:string} {0} to contain....`. This allows wrapping this call in an `AssertionScope` (and an upcoming feature that will try to get the `[CallerMemberName]`)